experiment/full.sh

```diff
@@ -617,5 +617,31 @@ skip_measurement=true ap sosreporter-gatherer.log \
   -e "sosreport_gatherer_local_dir='../../$logs/sosreport/'" \
   playbooks/satellite/sosreport_gatherer.yaml
 
+#AK Deletion
```

Comment:
Maybe add a new section?
```suggestion
section 'Deleting'

#AK Deletion
```